RecoLocalMuon/MuonSysAging/python/CSCChamberMasker_cfi.py

```diff
@@ -0,0 +1,11 @@
+
+import FWCore.ParameterSet.Config as cms
+
+CSCChamberMasker = cms.EDProducer('CSCChamberMasker',
```

Comment:
OK, will take care of this at the next iteration (when I understand where I should locate the package)